test/core/security/tls_security_connector_test.cc

```diff
@@ -1089,6 +1127,38 @@ TEST_F(TlsSecurityConnectorTest,
   core_external_verifier->Unref();
 }
 
+TEST_F(TlsSecurityConnectorTest,
+       ServerSecurityConnectorWithVerifiedRootSubjectCertSucceeds) {
+  auto* sync_verifier = new SyncExternalVerifier(true);
+  ExternalCertificateVerifier core_external_verifier(sync_verifier->base());
+  RefCountedPtr<grpc_tls_credentials_options> options =
+      MakeRefCounted<grpc_tls_credentials_options>();
+  options->set_cert_request_type(GRPC_SSL_DONT_REQUEST_CLIENT_CERTIFICATE);
```

Comment:
I'm guessing it doesn't make a big difference to the test, but this option should be set to something different, since if the server is not requesting the client cert then it doesn't make sense to get the verified root.

Can we just remove this line entirely? If it is necessary, can we change it to request and require verify?

Reply:
Changed to request and require verify